mcs/tools/mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -2346,41 +2346,29 @@ private void MakeAttributes (XmlElement root, IEnumerable<string> attributes, Ty
 	}
 
 	public static string MakeAttributesValueString (object v, TypeReference valueType)
-	{
-		if (v == null)
-			return "null";
-			if (valueType.FullName == "System.Type") {
-				var vTypeRef = v as TypeReference;
-				if (vTypeRef != null)
-					return "typeof(" + NativeTypeManager.GetTranslatedName (vTypeRef) + ")"; // TODO: drop NS handling
-				else
-					return "typeof(" + v.ToString () + ")";
+		{
+			// array: flagsformatter, appleformatter, standardflags, default
+			// foreach -> tryformat
+			var formatters = new [] { 
+				new AttributeValueFormatter (), 
+				new ApplePlatformEnumFormatter (), 
+				new StandardFlagsEnumFormatter (), 
+				new DefaultAttributeValueFormatter () 
```

Comment:
Whenever possible (e.g. arrays, dictionaries), use a trailing `,`. This will allow you to append new values without needing to append a `,` to the previously-last value.